_pages/en/strategy-technology-management.md

```diff
@@ -0,0 +1,338 @@
+    It is to establish foundational courses and skills needed to manage modern software effectively, securely, that benefits Canadians, and necessary to <a href="https://derisking-guide.18f.gov/">deliver successful technology projects</a>.</p>
+    <p>Acknowledging the government's current financial constraints, the Literacy Program will seek to leverage existing materials and services such as:</p>
+    <ul>
+    <li>The Linux Foundation's (e.g. <a href="https://training.linuxfoundation.org/training/introduction-to-devsecops-for-managers-lfs180x/">Introduction to DevSecOps for Managers</a>)</li>
+    <li>Institute for Citizen-Centered Services (e.g. <a href="https://iccs-isac.github.io/docs/open_source/">Open Source Software for Public Sector Executives</a>)</li>
+    <li>Canada School of Public Service</li>
+    <li>Public Cloud Service Providers</li>
+    <li>TBS Digital Talent Program</li>
+    </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="action-header"><strong>2. Adopt Product Management for all APM applications</strong></td>
+    <td rowspan="2">(removed)</td>
+  </tr>
+  <tr>
+    <td>
+<p>Technologies are just tools and remain relevant as long as their owners (“The Business”) continuously invest in it.
+This investment is not driven by the IT organization but by the business needs.
+Technologies hardware component may deteriorate over time but their software components need to constantly be worked on.
+Temporary Projects become one of the many sources of requirements that inform a software product's roadmap over its life cycle.
+Product Management <a href="https://014gc.sharepoint.com/:p:/r/sites/IITB-ITStrategy/Shared%20Documents/STRATEGIES%20AND%20INITIATIVES/Product%20Management/Presentations/Pilot%20Product%20Management%20Oversight%20Framework-EMC-v4-EN.pptx?d=w94fa7388e6db445ca550d4e191c65a72&csf=1&web=1&e=WYUJvd">mitigates the risk of application obsolescence</a> by keeping them relevant and, as such, ensures ESDC Program services maintains pace of relevance with Canadian expectations.</p>
+
+<p>Adopting Product Management requires a different instrument to formalize multi-disciplinary teams outside of temporary projects between “The Business” and IT to enable the sustainment and continuously improvements of APM applications used in Programs service delivery.</p>
+
+<p>Authorities to establish such instrument at ESDC is at CIO and CFO level.</p>
+    </td>
+  </tr>
+  <tr>
+    <td class="action-header"><strong>3. Implement DevSecOps</strong></td>
+    <td rowspan="2">(removed)</td>
+  </tr>
+  <tr>
+    <td>
+<p>Inspired by the <a href="https://software.af.mil/team/platformone/">US Air Force's Platform One team</a>, DevSecOps focuses on SecOps (security at runtime) and <i>incentivizes</i> re-using existing software components and applications instead of proliferating new ones (the #1 help for IT Security).</p>
+
+<p>Architecture and release management require to adapt to realize the benefits to DevSecOps.</p>

```

Comment:
Should this be "Architecture and release management are required to adapt to realize...."